Library/Source/NativeEngine.cpp

```diff
@@ -92,7 +96,7 @@ namespace babylon
 
                 if (memberType.basetype != spirv_cross::SPIRType::Float)
                 {
-                    throw std::exception("Not supported");
+                    throw std::exception();
```

Comment:
Should we consider replacing this with a babylon::exception{} type that inherits from std::exception and has the behavior we'd been using, sort of like is done in the example in the docs?

http://www.cplusplus.com/reference/exception/exception/exception/

Reply:
Yes, that's the way to go